amr-wind/wind_energy/ABLBoundaryPlane.cpp

```diff
@@ -0,0 +1,751 @@
+#include "netcdf.h"
```

Comment:
should we ifdef NetCDF? as done here: https://github.com/Exawind/amr-wind/blob/af70ab0decf4f6b353f5c0d63d5e303b63ba0798/amr-wind/utilities/ncutils/nc_interface.H#L12

Reply:
I asked @sayerhs about that... He was of the opinion we should just make NetCDF a non-optional dependency since this is a capability we probably want turned on most of the time.